cmd/imagec/imagec.go

```diff
@@ -262,24 +266,28 @@ func ImagesToDownload(manifest *Manifest, storeName string) ([]*ImageWithMeta, e
 
 	// return early if -standalone set
 	if options.standalone {
-		return images, nil
+		return images, nil, nil
 	}
 
 	// Create the image store just in case
 	err := CreateImageStore(storeName)
```

Comment:
We should restructure this, and anything similar, so that it's triggered as remediate/retry based on error returns when time allows. Not for this PR.